libvirt/tests/src/migration/migration_with_disk/migration_with_vhostuser.py

```
@@ -0,0 +1,198 @@
+    Start one vhost sock service in source host.
+
+    :param start_sock_service_cmd: command to start vhost service
+    :param image_path: image file path
+    :param sock_path: sock file path
+    """
+    # Create backend image in source host
+    libvirt.create_local_disk("file", image_path, size="100M")
+    chown_cmd = "chown qemu:qemu %s" % image_path
+    process.run(chown_cmd, ignore_status=False, shell=True)
+    # Start vhost sock service in source host
+    cmd_output = process.run(start_sock_service_cmd, ignore_status=False, shell=True).stdout_text.strip()
```

Ruff (0.14.6)

Local variable `cmd_output` is assigned to but never used

Remove assignment to unused variable `cmd_output`

(F841)